protected/humhub/modules/comment/models/Comment.php

```diff
@@ -220,7 +221,11 @@ public static function GetCommentsLimited($model, $id, $limit = null)
 
             $query = Comment::find();
             $query->offset($commentCount - $limit);
-            $query->orderBy('created_at ASC');
+            if (Comment::findOne(['id' => $currentCommentId])) {
+                $query->orderBy('`comment`.`id` <= ' . ($currentCommentId + $limit - 1));
```

Comment:
Can you please additionally enfore integers? Maybe intval($currentCommentId) && intval($limit). Just to be paranoid ;-)

Reply:
Done here https://github.com/humhub/humhub/pull/5168/commits/fec4ce7beb1f04b67002ab501037f8e3c5ff260f#diff-4fcd54238adfe8774ee607eeadd9ff1c821166188d5f9dabd37cac4e2729a218R225.